README.md

```diff
@@ -72,9 +72,9 @@ Create the database tables running:
 
 ## Configuration
 
-Once installed, add the `validation` plugin to the `ckan.plugins` configuration option in your INI file:
+Once installed, add the `validation` plugin to the `ckan.plugins` configuration option in your INI file. If using ckanext-scheming, the `validation` plugins should be loaded **before** the `scheming_datasets` one:
```

Comment:
Is it feasible to remove this constraint? I know that plugin ordering can be important, but perhaps there is a way to make them more independent?

Reply:
@ThrawnCA I don't think so. We want ckanext-validation's [errors.html](https://github.com/frictionlessdata/ckanext-validation/blob/master/ckanext/validation/templates/scheming/snippets/errors.html) to override ckanext-scheming's so we can hook the report to the error messages.

I guess that we could add "If using ckanext-scheming and using synchronous mode, the `validation` plugin should be loaded **before** the `scheming_datasets` one" but that would make the instructions a bit more confusing

Reply:
Oh, and ckanext-scheming defines the base version of that template, which doesn't extend anything, so if ckanext-scheming comes first, then processing stops and ckanext-validation's version is never reached. That makes sense.